src/console/Startup.cs

```diff
@@ -188,5 +194,28 @@ public string GetConsoleApplicationType()
 
             return applicationType;
         }
+
+        private void AddContentTemplateFactories(IServiceCollection services)
+        {
+            services.AddSingleton<IExpressionRegister>(sp => new AssemblyExpressionRegister(typeof(IExpressionRegister).Assembly, sp.GetRequiredService<ITelemetryLogger>()));
+            services.AddSingleton<IExpressionEvaluatorFactory>(
+                sp =>
+                {
+                    var jmesPath = new JmesPath();
```

Comment:
>var jmesPath = new JmesPath(); [](http://example.com/codeflow?start=20&length=30)

I think a better approach here would be to define the singleton for building the JmesPath that includes setting the expressionRegister, and using normal DI for the jmesPath object for the TemplateExpressionEvaluatorFactory

Reply:
Main weakness here is if anything else needs the JmesPath object in the future it won't properly instantiated.